onnxruntime/core/providers/tensorrt/onnx_ctx_model_helper.cc

```diff
@@ -236,6 +244,19 @@ bool IsRelativePathToParentPath(std::string& path_string) {
 #endif
 }
 
+// Get the refitted engine cache path
+std::string GetRefittedEnginePath(std::string engine_cache) {
+  std::filesystem::path engine_cache_path(engine_cache);
+  // The weight-stripped engine has the naming of xxx.stripped.engine
```

Comment:
i'm not sure what this comment line 250 is trying to tell us.

Reply:
i get a little confused by the terminology refitted engine.  it's just engine and stripped.engine right?
There's no real difference between a regular engine that has weights and an engine that was refitted?

Reply:
> i get a little confused by the terminology refitted engine. it's just engine and stripped.engine right? There's no real difference between a regular engine that has weights and an engine that was refitted?

Yes, that's all correct

Reply:
i added the comment and updated the function name for this.